ee/tabby-db/src/invitations.rs

```diff
@@ -60,27 +61,32 @@ impl DbConn {
             return Err(anyhow!("User already registered"));
         }
 
+        let mut transaction = self.pool.begin().await?;
         let code = Uuid::new_v4().to_string();
         let res = query!(
             "INSERT INTO invitations (email, code) VALUES (?, ?)",
```

Comment:
```suggestion
            "INSERT INTO invitations (email, code, created_at) VALUES (?, ?, ?)",
```

You can insert created_at with `system::now` to avoid an additional query

Reply:
Tried this, but the issue is that `created_at` is a string, whose formatting is dependent on SQLite. I wanted to ensure that the format will be the same when you're creating vs requesting the invitation.

Reply:
created_at is not a string, it's an timestamp in db.

the DAO defines it wrongly, take a look at https://github.com/TabbyML/tabby/blob/009f467902bed56b3406a5593b94cb23cd1504fd/ee/tabby-db/src/users.rs#L11 for example

Reply:
It's a string in InvitationDAO, I can change it to be DateTime instead.